src/common/python/constants.py

```diff
@@ -34,6 +36,9 @@
 WAITER_FAILURE = "Waiter encountered a terminal failure state"
 RATE_EXCEEDED = "Rate exceeded"
 WAITER_MAX = "Max attempts exceeded"
+EXPORT_FROM_REGION_THAT_SUPPORTS_SNAPSHOT_EXPORT_TO_S3 = "export_from_region_that_supports_snapshot_export_to_s3_task"
```

Comment:
please convert all the namings to be more concise

Reply:
@namitad I am all for being as concise as possible to the point where you don't loose information. The result is what you see. I don't believe it can be shorter. Can you give me an example without loosing the essence? Otherwise let's keep it as it is.

Reply:
Again :) ["2 hard things in computer science caching and naming things."](https://martinfowler.com/bliki/TwoHardThings.html) :) but ok, let me try to clarify and we maybe find something more convenient to all. 

- "Ideally are we saying we are exporting from a region or exporting to a region"?  
It is more "from" than "to", but perhaps more appropriate is "in" (both the snapshot and the s3 have to be in the same region). It refers to (the full name would be) EXPORT_SNAPSHOT_TO_S3_IN_REGION_THAT_SUPPORTS_SNAPSHOT_EXPORT_TO_S3 . 

- "Also your LHS and RHS looks almost the same which is concerning."
Ok, will change the RHS to "Case of exporting snapshot to s3 in a region that supports snapshot export to s3."

- "Can we try something of the sort "EXPORT_SNAPSHOT_TO_SUPPORTED_REGION" which i feel is more comprehensible."
No - kindly check the above 2 points. 

Does this make sense @namitad @stationeros ? Let me know if still concerns.